database/layer/knex.js

```diff
@@ -1,16 +1,31 @@
-const knex = require('knex');
-const knexDbConfig = require('../knexfile');
+const knex = require('knex')
+const knexDbConfig = require('../knexfile')
 
 /**
  * Usher DB connection instance.
  * @name usherDb
  * @type {import('knex')}
  * @desc This instance provides a connection to the Usher database using Knex.js
  * @example // To import usherDb instance
- * const { usherDb } = require('./knex');
+ * const { usherDb } = require('./knex')
  * @example // To perform a database query
- * const persona = await usherDb('personas').where('key', personaKey).first();
+ * const persona = await usherDb('personas').where('key', personaKey).first()
  */
-const usherDb = knex(knexDbConfig);
+let usherDb
+try {
+  usherDb = knex(knexDbConfig)
+  if (usherDb?.client?.pool) {
+    const pool = usherDb.client.pool
+    pool.idleTimeoutMillis = 0
+
+    pool.on('release', () => {
```

Comment:
@k1ch should we add a comment here about what/why this logic is doing? ie. is this related to functionality in tarn that will help ensure releasing connections.

Reply:
Sure, I can add an additional comment. This logic ensures that idle connections are released immediately. The `idleTimeoutMillis` setting cannot be configured through Knex, so we override it directly.